graylog/resource/user/resource.go

```diff
@@ -36,8 +36,23 @@ func Resource() *schema.Resource {
 				Required: true,
 			},
 			"full_name": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				Required:      false,
+				Computed:      true,
+				ConflictsWith: []string{"first_name", "last_name"},
+			},
+			"first_name": {
```

Comment:
Could you updated the docs for this resource to add the new `first_name` & `last_name`, as well as mentioning they conflict with `full_name`?
Docs file can be found here: `docs/resources/user.md`

Reply:
Sure, done

Reply:
👍